### PR TITLE
gnupg 2.1.4 -> 2.1.5

### PR DESCRIPTION
**Pull request by DamienCassou**
According to someone on #gnupg, the patch is not necessary anymore in the 2.1 series. Also, the patch does not apply.

@wkennington @shlevy what do you think?

### CODE REVIEW

**Comment by domenkozar** [comment]
The patch was submitted by someone via ML

**Comment by DamienCassou** [comment]
@domenkozar so?

File: pkgs/tools/security/gnupg/21.nix [hunk 1]
```diff
@@ -13,15 +13,13 @@ with stdenv.lib;
...
-  name = "gnupg-2.1.4";
+  name = "gnupg-2.1.5";
 
   src = fetchurl {
     url = "mirror://gnupg/gnupg/${name}.tar.bz2";
-    sha256 = "1c3c89b7ziknz6h1dnwmfjhgyy28g982rcncrhmhylb8v3npw4k4";
+    sha256 = "0k5818r847zplbrwjp6i48s6xb5zy44rny2kmbisd6y3c1qml45m";
   };
 
-  patches = [ ./socket-activate-2.1.1.patch ];
```
Comment (pSub) [inline pkgs/tools/security/gnupg/21.nix]: You should also remove the patch if it isn't needed anymore.

Comment (DamienCassou) [inline pkgs/tools/security/gnupg/21.nix]: done

**Comment by shlevy** [comment]
Yeah, the patch was rejected upstream, best to just remove it.

**Pull request merged**